src/test/java/webserver/HttpRequestTest.java

```diff
@@ -0,0 +1,70 @@
+package webserver;
+
+import model.StartLine;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+class HttpRequestTest {
+
+    private static final String PATH = "src/main/resources/templates";
+
+    @Test
+    @DisplayName("요청 메시지의 start-line 에서 요청 메서드를 파싱할 수 있다.")
```

Comment:
```suggestion
    @DisplayName("Request 의 start-line에 포함된 GET 메소드를 파싱해서 리턴한다.")
```
딴 생각하다 이걸 왜 썼는지 이유를 잊었습니다. 쓴 게 아까워서 남겨둡니다...